proposals/NNNN-New-vehicle-data-GearStatus.md

```diff
@@ -0,0 +1,230 @@
+		<description>Tells the actual gear selected by transmission</description>
+	</param>
+</struct>
+```
+#### Add new enum `GearPosition`:
+
+```xml	
+<enum name="GearPosition" since="x.x">
+	<description>The selected gear.</description>
+	<element name="PARK">
+		<description>Parking</description>
+	</element>
+	<element name="REVERSE">
+		<description>Reverse gear</description>
+	</element>
+	<element name="NEUTRAL">
+		<description>No gear</description>
+	</element>
+	<element name="DRIVE">
+	</element>
+	<element name="SPORT">
+		<description>Drive Sport mode</description>
+	</element>
+	<element name="LOWGEAR">
+		<description>1st gear hold</description>
+	</element>
+	<element name="FIRST">
+	</element>
+	<element name="SECOND">
+	</element>
+	<element name="THIRD">
+	</element>
+	<element name="FOURTH">
+	</element>
+	<element name="FIFTH">
+	</element>
+	<element name="SIXTH">
+	</element>
+	<element name="SEVENTH">
+	</element>
+	<element name="EIGHTH">
+	</element>
+	<element name="NINTH">
+	</element>
+	<element name="TENTH">
+	</element>
+	<element name="UNKNOWN">
+	</element>
+	<element name="FAULT">
+	</element>
+</enum>	
+```
+#### Add the following parameter to these function requests:
+* `SubscribeVehicleData`
+* `UnsubscribeVehicleData`
+* `GetVehicleData`
+
+```xml	
+<param name="gearStatus" type="Boolean" mandatory="false" since="X.x">
+	<description>See GearStatus</description>
+</param>
+```
+
+#### Add the following parameter to these function responses:
+* `SubscribeVehicleData`
+* `UnsubscribeVehicleData`
+
+```xml	
+<param name="gearStatus" type="VehicleDataResult" mandatory="false" since="X.x">
+	<description>See GearStatus</description>
+</param>
+```
+
+#### Add the following parameter to these function responses:
+* `GetVehicleData`
+* `OnVehicleData`
+
+```xml	
+<param name="gearStatus" type="GearStatus" mandatory="false" since="X.x">
+	<description>See GearStatus</description>
+</param>
+```
+
+### Updates in HMI_API 
+
+#### Add to enum `VehicleDataType` in `Common` interface: 
+
+```xml	
+<element name="VEHICLEDATA_GEARSTATUS"/>
+```
+#### Add new struct `GearStatus` in `Common` interface:
+
+```xml	
+<struct name="GearStatus">
+	<param name="gearDriveMode" type="Common.GearPosition" mandatory="false">
+		<description>Tells the gear drive mode i.e. PARK, DRIVE, SPORT etc</description>
+	</param>
+	<param name="actualGear" type="Common.GearPosition" mandatory="false">
+		<description>Tells the actual gear selected by transmission</description>
+	</param>
+</struct>
+```
+#### Add new enum `GearPosition` in `Common` interface:
+
+```xml	
+<enum name="GearPosition">
+	<description>The selected gear.</description>
+	<element name="PARK">
+		<description>Parking</description>
+	</element>
+	<element name="REVERSE">
+		<description>Reverse gear</description>
+	</element>
+	<element name="NEUTRAL">
+		<description>No gear</description>
+	</element>
+	<element name="DRIVE">
+	</element>
```

Comment:
Should there be a description included here?

Reply:
Added the description